src/main.go

```diff
@@ -61,6 +61,18 @@ func main() {
 		updateNextNode,
 	)
 
+	localIP, err := network.LocalIP()
+
+	if err != nil {
+		fmt.Println(err)
+		localIP = "DISCONNECTED"
+	}
+
+	incomingMessageChannel := make(chan []byte)
+
+	go network.ListenForMessages(localIP, 8080, incomingMessageChannel)
```

Comment:
The port we listen on must be the same as the broadcast port (from the elev state).